.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: ruby
+
+services:
+  - mysql
+
+before_script:
+  - cp config/database.yml.sample config/database.yml
```

Comment:
Maybe the example database file can be removed in future commits, when the database configuration is more clear.